doc/Development_Documentation/21_Deployment/03_Configuration_Environments.md

```diff
@@ -38,4 +38,5 @@ Available environment variables are:
 PIMCORE_WRITE_TARGET_IMAGE_THUMBNAILS=settings-store
 PIMCORE_WRITE_TARGET_CUSTOM_REPORTS=settings-store
 PIMCORE_WRITE_TARGET_VIDEO_THUMBNAILS=settings-store
+PIMCORE_WRITE_TARGET_WEB2PRINT=settings-store
```

Comment:
Can't add a suggestion, but this should be added in the list above (line 27) too, see: #10217